jpo-asn-j2735-2024/README.md

```diff
@@ -0,0 +1,58 @@
+# J2735/2024 POJOs
+
+This is a Java library with the 2024 POJOs.
+
+To build:
+
+```bash
+./gradlew clean build
+```
+Tools for generating test data are in the

```

Comment:
question: why do we need to generate test data in this manner? Can we explain that a bit more here?

Reply:
Added a "Testing" section to that README describing how to run the tests and why generating test data was necessary.